src/content/app/genome-browser/components/drawer/components/sequence-view/DrawerSequenceView.tsx

```diff
@@ -0,0 +1,186 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState, useMemo, useEffect } from 'react';
+import classNames from 'classnames';
+
+import { getReverseComplement } from 'src/shared/helpers/sequenceHelpers';
+
+import RadioGroup from 'src/shared/components/radio-group/RadioGroup';
+import Checkbox from 'src/shared/components/checkbox/Checkbox';
+import ShowHide from 'src/shared/components/show-hide/ShowHide';
+
+import type { SequenceType } from 'src/content/app/genome-browser/state/drawer/drawer-sequence/drawerSequenceSlice';
+
+import styles from './DrawerSequenceView.scss';
+
+const sequenceLabelsMap: Record<SequenceType, string> = {
+  genomic: 'Genomic sequence',
+  cdna: 'cDNA',
+  cds: 'CDS',
+  protein: 'Protein sequence'
+};
+
+// TODO: we probably also want to pass a sequence header in order to be able to blast it
+type Props = {
+  isExpanded: boolean;
+  toggleSequenceVisibility: () => void;
+  sequence?: string;
+  sequenceTypes: SequenceType[];
+  selectedSequenceType: SequenceType;
+  isReverseComplement: boolean;
+  onSequenceTypeChange: (sequenceType: SequenceType) => void;
```

Comment:
Can't this be optional instead of requiring to pass `noop` when we don't want to pass anything?

Reply:
The `RadioGroup` component requires an `onChange` handler. This is what the `onSequenceTypeChange` is here. I don't think it makes sense to make the `onChange` of the `RadioGroup` optional. I think the noop for this particular case is a better option.